node/npm_dependencies.ts

```diff
@@ -110,6 +110,7 @@ const getNPMSpecifiers = async (
     readFile: async (filePath: string) => {
       // If this is a TypeScript file, we need to compile in before we can
       // parse it.
+      // mabye we should make this `path.extname(filepath).includes("ts")`?
       if (TYPESCRIPT_EXTENSIONS.has(path.extname(filePath))) {
```

Comment:
mabye we should make this `path.extname(filepath).includes("ts")`?

Reply:
Why?

Reply:
it wouldn't require us to update the set everytime we learn about a new typescript extension ;D happy to keep it how it is, though